CONTRIBUTING.md

```diff
@@ -22,3 +57,43 @@ tox -e integration   # integration tests
 tox                  # runs 'lint' and 'unit' environments
 ```
 
+
+## Build charm
+
+Build the charm in this git repository using:
+
+```shell
+cd mongodb-operator/
+charmcraft pack
+```
+
+### Deploy
```

Comment:
Could this be a typo on the template? https://github.com/canonical/operator-template/blob/main/CONTRIBUTING.md

Reply:
I think so, but also I didn't write the template and I'm not sure who to ask about it 🤷 

either way I think this looks cleaner and it will have consistent format with the other sections